pkg/virt-api/rest/authorizer.go

```diff
@@ -138,25 +138,6 @@ func (a *authorizor) generateAccessReview(req *restful.Request) (*authorization.
 		return nil, fmt.Errorf("no URL in http request")
 	}
 
-	// URL example
```

Comment:
I'd prefer to see these authorizer changes in another commit to the rename but only if you need to respin the series.